cloud/aws-functions/orphaned_cloudformation.py

```diff
@@ -7,40 +7,53 @@
 from utils import get_regions_list
 from time import sleep
 
+
 def is_stack_to_terminate(stack, aws_region):
-    cf_client = boto3.client('cloudformation', region_name=aws_region)
+    cf_client = boto3.client("cloudformation", region_name=aws_region)
 
     try:
-        stack_desc = cf_client.describe_stacks(StackName=stack)['Stacks'][0]
-        tags = stack_desc['Tags']
+        stack_desc = cf_client.describe_stacks(StackName=stack)["Stacks"][0]
+        tags = stack_desc["Tags"]
 
     except ClientError as e:
         print(e)
         return False
 
-    tags_dict = {item['Key']: item['Value'] for item in tags}
+    tags_dict = {item["Key"]: item["Value"] for item in tags}
 
-    if 'team' not in tags_dict.keys() or ('team' in tags_dict.keys() and tags_dict['team'] != 'cloud'):
+    if "team" not in tags_dict.keys() or (
+        "team" in tags_dict.keys() and tags_dict["team"] != "cloud"
+    ):
```

Comment:
We can simplify this conditional with:
`if cluster["cluster"]["tags"].get("team") != "cloud"`
Please check other files as well